sdk/attestation/Azure.Security.Attestation/src/Generated/PolicyCertificatesRestClient.cs

```diff
@@ -54,6 +54,7 @@ internal HttpMessage CreateGetRequest()
 
         /// <summary> Retrieves the set of certificates used to express policy for the current tenant. </summary>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
+        /// <remarks> Retrieves the set of certificates used to express policy for the current tenant. </remarks>
```

Comment:
This doesn't look right to me.   The summary and remarks shouldn't be repeated.   Not an issue for an internal client type such as this, but it would be a concern for a pure DPG client.

Reply:
If we change the M4 behavior, it would potentially be a breaking change with a wider impact.   While I, personally, disagree with the M4 behavior, it's a safer change to include this downstream since these are new changes.